app/pages/project/instances/instance/tabs/StorageTab.tsx

```diff
@@ -174,7 +174,12 @@ export function StorageTab() {
           <Button
             size="sm"
             onClick={() => setShowDiskCreate(true)}
-            disabledReason={<>Instance must be {attachableStates} to create a disk</>}
+            disabledReason={
+              <>
+                Instance must be <span className="text-default">stopped</span> to create a
+                disk
+              </>
+            }
```

Comment:
could it be worth saying “create and attach” here? it’s the attach that’s the problem, right?